samples/rest-server/datauser/credentialapis.go

```diff
@@ -66,24 +69,23 @@ func GetCredentials(w http.ResponseWriter, r *http.Request) {
 }
 
 // DeleteCredentials deletes the secret
-//nolint:dupl
 func DeleteCredentials(w http.ResponseWriter, r *http.Request) {
 	log.Println("In DeleteCredentials")
 	if k8sClient == nil {
 		suberr := render.Render(w, r, ErrConfigProblem(errors.New("no client set")))
 		if suberr != nil {
-			log.Printf(suberr.Error() + " upon no client set")
+			log.Print(suberr.Error())
 		}
 	}
 
-	secretName := chi.URLParam(r, "secret")
+	secretName := chi.URLParam(r, paramSecret)
 
 	// Call kubernetes to get the FybrikApplication CRD
 	err := k8sClient.DeleteSecret(secretName, nil)
 	if err != nil {
 		suberr := render.Render(w, r, ErrRender(err))
 		if suberr != nil {
-			log.Printf(suberr.Error() + " upon " + err.Error())
+			log.Printf(suberr.Error() + " : " + err.Error())
 		}
```

Comment:
When a REST call returns without setting an HTTP error, what should be returned to the client?
should we set something like `render.Status(r, http.StatusInternalServerError)`

Reply:
Fixed